Simplenote.xcodeproj/project.pbxproj

```diff
@@ -84,7 +84,7 @@
 		37D4DD6A20B3574D00C225EA /* WPAuthHandler.m in Sources */ = {isa = PBXBuildFile; fileRef = 37D4DD6820B3574C00C225EA /* WPAuthHandler.m */; };
 		37F742EB202A382400A47D3A /* AboutViewController.swift in Sources */ = {isa = PBXBuildFile; fileRef = 37F742EA202A382400A47D3A /* AboutViewController.swift */; };
 		37F742EC202A382400A47D3A /* AboutViewController.swift in Sources */ = {isa = PBXBuildFile; fileRef = 37F742EA202A382400A47D3A /* AboutViewController.swift */; };
-		3FAC7B25254BBCA600B47276 /* SimplenoteInterlinks in Frameworks */ = {isa = PBXBuildFile; productRef = 3FAC7B24254BBCA600B47276 /* SimplenoteInterlinks */; };
```

Comment:
Update. Of course, now that I said "Xcode will revert it at some point" I'm not able to trigger that change 😳 .

I still think this is incorrect. Here's two data points in favor of this hypothesis:

- [Tuist](https://tuist.io/, an open source Xcode project generator, has [a type for `XCRemoteSwiftPackageReference`](https://github.com/tuist/XcodeProj/blob/82bf5efcaa27e94ed8c761c1eb3e397b6dea82b9/Sources/XcodeProj/Objects/SwiftPackage/XCRemoteSwiftPackageReference.swift) but no type for `RemoteSwiftPackageReference`
- Someone asked something similar to this [on StackOverflow](https://stackoverflow.com/questions/59830330/swiftlint-overriding-project-settings-related-to-spm)

Reply:
So, I can confirm that the format that Xcode uses to generate the pbxproj might sometimes slightly differ from what CocoaPods, Tuist, XcodeGen or others generate, because Xcode's `pbxproj` generator is private implementation of course…

The usual recommendation I'd do is indeed to modify something in the Xcode project (e.g. move a file in one of the groups in the file navigator to reorder the files, then move it back to its original position) then save the project via Xcode UI to restore what Xcode generates at the end. No real magic solution for this unfortunately.

Reply:
Nice fix - TIL about that trick.